<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputNO2Form</class>
 <widget class="QWidget" name="OutputNO2Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>277</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>277</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Outputting NO2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NO2ValueLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NO2UnitsLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ppb</string>
   </property>
  </widget>
  <widget class="QLabel" name="NOsetpointLabel_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>NO2 Output:</string>
   </property>
  </widget>
  <widget class="QLabel" name="NOValueLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>110</y>
     <width>100</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NOsetpointLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>200</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>NO Output:</string>
   </property>
  </widget>
  <widget class="QLabel" name="NOUnitsLabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>110</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ppb</string>
   </property>
  </widget>
  <widget class="QLabel" name="noLightIconLabel">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>151</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="noFlowIconLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>151</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="noTempIconLabel">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>151</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>58</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>NOChannel:</string>
   </property>
  </widget>
  <widget class="QLabel" name="o3FlowIconLabel">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>181</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>180</y>
     <width>57</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>O3Channel:</string>
   </property>
  </widget>
  <widget class="QLabel" name="o3LightIconLabel">
   <property name="geometry">
    <rect>
     <x>296</x>
     <y>181</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="o3TempIconLabel">
   <property name="geometry">
    <rect>
     <x>222</x>
     <y>181</y>
     <width>20</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>105</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>105</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scrollRightButton">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>220</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="scrollLeftButton">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>220</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="diagnosticData">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>235</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>235</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Diagnostics</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="TitleBackground" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <zorder>TitleBackground</zorder>
  <zorder>Title</zorder>
  <zorder>NO2ValueLabel</zorder>
  <zorder>NO2UnitsLabel</zorder>
  <zorder>NOsetpointLabel_2</zorder>
  <zorder>NOValueLabel</zorder>
  <zorder>NOsetpointLabel</zorder>
  <zorder>NOUnitsLabel</zorder>
  <zorder>noLightIconLabel</zorder>
  <zorder>noFlowIconLabel</zorder>
  <zorder>noTempIconLabel</zorder>
  <zorder>label</zorder>
  <zorder>o3FlowIconLabel</zorder>
  <zorder>label_2</zorder>
  <zorder>o3LightIconLabel</zorder>
  <zorder>o3TempIconLabel</zorder>
  <zorder>quitButton</zorder>
  <zorder>scrollRightButton</zorder>
  <zorder>scrollLeftButton</zorder>
  <zorder>diagnosticData</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
